hips/draw/paint.py

```diff
@@ -221,6 +221,30 @@ def plot_mpl_hips_tile_grid(self) -> None:
                     transform=ax.get_transform('world'), **opts)
         ax.imshow(self.image, origin='lower')
 
+    def plot_mpl_hips_tile_number_grid(self) -> None:
```

Comment:
I thought we already had such a helper method?

Suggest to either make a separate PR for additions to the hips package, or to just move this to the `plot_jpg.py` script. (let's not mix package additions / changes with this paper, e.g. the paper might be merged after v0.3).

Reply:
Ah, this was committed by mistake. The current function only drew grid lines around the tiles. I was trying to add the tile numbers to it, similar to what [Aladin Lite](http://aladin.unistra.fr/AladinLite/) does (with HEALPix grid turned on).

Once this functionality is complete, I think this can be merged with the current function `plot_mpl_hips_tile_grid` (in a separate PR).